src/main/java/com/minecolonies/coremod/util/ChunkDataHelper.java

```diff
@@ -334,11 +339,21 @@ public static void claimChunksInRange(
                 {
                     continue;
                 }
+                else
+                {
+                    areAllChunksAdded = false;
```

Comment:
That var seems a bit confusing, might need a better name. Maybe "wereChunksAdded" ?

Reply:
Do we need this var at all?

Reply:
Yes because of the nested loop.